stockAndPrices/services/app/templates/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> Home Page</title>
+    </head>
+
+    <body>
```

Comment:
We can leave these html files in for now, but remove them once the react project is hooked up

Reply:
Removed the two html files and the templates folder.